### PR TITLE
Fix canvas context leaks

### PR DESCRIPTION
**Pull request by MrMeison**

### CODE REVIEW

File: pbiviz.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
   "visual": {
     "name": "WordCloud",
     "displayName": "WordCloud",
-    "guid": "WordCloud1447959067750",
+    "guid": "WordCloud1447959067751",
```
Comment (ignatvilesov) [inline pbiviz.json]: Why did you change it?

File: package.json [hunk 2]
```diff
@@ -29,30 +29,32 @@
   "homepage": "https://github.com/Microsoft/PowerBI-visuals-WordCloud#readme",
   "dependencies": {
     "d3": "3.5.5",
-    "lodash": "4.16.2",
-    "powerbi-visuals-utils-colorutils": "0.2.1",
-    "powerbi-visuals-utils-dataviewutils": "1.0.1",
-    "powerbi-visuals-utils-formattingutils": "0.2.2",
-    "powerbi-visuals-utils-svgutils": "0.2.1",
-    "powerbi-visuals-utils-typeutils": "0.2.1"
+    "d3-cloud": "1.2.4",
```
Comment (ignatvilesov) [inline package.json]: Do you use it?

File: package.json [hunk 3]
```diff
@@ -29,30 +29,32 @@
...
+    "lodash": "4.17.4",
+    "powerbi-visuals-utils-colorutils": "^1.0.0",
+    "powerbi-visuals-utils-dataviewutils": "^1.0.0",
+    "powerbi-visuals-utils-formattingutils": "^1.0.0",
+    "powerbi-visuals-utils-svgutils": "^1.0.0",
+    "powerbi-visuals-utils-typeutils": "^1.0.0"
   },
   "devDependencies": {
     "@types/d3": "3.5.36",
+    "@types/d3.cloud.layout": "^1.2.32",
```
Comment (ignatvilesov) [inline package.json]: Do you use it?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-wordcloud",
-  "version": "1.2.11",
+  "version": "1.2.12",
```
Comment (ignatvilesov) [inline package.json]: Please update version in pbiviz.json as well

Comment (ignatvilesov) [inline package.json]: Please include CHANGELOG

File: test/visualTest.ts [hunk 5]
```diff
@@ -247,8 +248,9 @@ module powerbi.extensibility.visual.test {
         describe("Format settings test", () => {
             describe("Data color", () => {
                 it("colors", (done) => {
-                    let category: DataViewCategoryColumn,
-                        colors: string[];
+                    const mockColorPallete = createColorPalette();
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please specify a type

File: pbiviz.json (hunk 1 above)
Comment (MrMeison) [inline pbiviz.json]: It was changed on update powerBI API version automatically
